cases/api_views.py

```
@@ -239,7 +239,7 @@ def get_queryset(self):
             # - Other authenticated (contributor): PUBLISHED + cases they are assigned to
             if not (self.request.user and self.request.user.is_authenticated):
                 queryset = Case.objects.filter(state=CaseState.PUBLISHED)
-            elif is_admin_or_moderator(self.request.user):
+            elif is_admin_or_moderator(self.request.user) or is_contributor(self.request.user):
```

Ruff (0.15.15)

[error] Undefined name `is_contributor`

(F821)